packages/manager/apps/telecom/src/app/telecom/pack/migration/confirm/pack-migration-confirm.controller.js

```diff
@@ -24,29 +24,48 @@ export default class TelecomPackMigrationConfirmCtrl {
     };
 
     this.PROMO_DISPLAY = PROMO_DISPLAY;
+    this.QUANTITY = QUANTITY;
 
     this.modemTransportPrice = 9.99;
 
     this.process = this.TucPackMigrationProcess.getMigrationProcess();
     this.choosedAdditionalOptions = this.TucPackMigrationProcess.getOptionsSelected();
 
-    const modemRental = this.process.selectedOffer.modemRental
-      ? this.process.selectedOffer.modemRental.value
-      : 0;
+    let gtrComfortSelected = 0;
+    this.choosedAdditionalOptions.forEach((option) => {
+      if (option.name.match(/^gtr_\d{1,2}m_/)) {
+        gtrComfortSelected = 1;
+      }
+    });
```

Comment:
**suggestion:** Can `break` out of loop after finding the first match.
```suggestion
    let gtrComfortSelected = this.choosedAdditionalOptions.some((option) => option.name.match(/^gtr_\d{1,2}m_/));
    }) ? 1 : 0;
```

Reply:
thank you, change made